### PR TITLE
Add stats command

### PR DESCRIPTION
**Pull request by xjahic**
If you want to test it yourself be sure to add these to .env:
```
MAX_OLD_STAT_GRAPH_IN_DAYS =
BOT_COMMANDS_CHANNEL_ID =
```
Also you should import data (user_survey_answer) to your localhost db so you won't end up with empty statistics
However I am more than happy to show you how it works

### CODE REVIEW

**Review by gabrielKerekes — CHANGES_REQUESTED** [review]
This is it for today. I got a little bit carried away, sorry 😄  I haven't yet tried to run it, I'll do that later.

File: src/cogs/public_commands.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from discord.ext import commands
+
+from stats import stats
+
+
+class PublicCommands(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(name="stats")
```
[the remark this answers is not in the document]
Comment (xjahic) [reply src/cogs/public_commands.py:10]: Renamed to `show-stats`

File: src/stats/stats.py [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+                )
+                make_percentage_graph(
+                    option_id, options, answers, stat_title, graph_size=graph_size
+                )
+
+                await send_graph_to_channel(ctx, option_id, stat_title)
+
+
+# i.e will return tuple (['muz', 'zena'], [15, 12]) as in 15 users answered as muz and 12 as zena
+def get_options_and_answers(survey_question_id):
```
[the remark this answers is not in the document]
Comment (xjahic) [reply src/stats/stats.py:56]: Ok

File: src/stats/stats.py [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+    for i in range(data.shape[0]):
+        pct = (data.Answers[i] / total_answers) * 100
+        percentage.append((round(pct, 2)))
+    return percentage
+
+
+def make_percentage_graph(
+    graph_file_name, options, answers, stat_title, graph_size=None
+):
+    data = pd.DataFrame({"Option": options, "Answers": answers})
```
[the remark this answers is not in the document]
Comment (xjahic) [reply src/stats/stats.py:85]: Good point, I removed it. Now it is just dictionary

**Comment by xjahic** [comment]
I made a fixup 1af184c7406638446305244d98c9001af2057b0c with almost all requested changes

**Comment by gabrielKerekes** [comment]
Looks good now! Awesome job! Please squash and merge! 🥳 🎈

**Pull request merged**